advanced/Scripts/list.sh

```diff
@@ -125,7 +122,7 @@ AddDomain() {
     list="$2"
 
     # Is the domain in the list we want to add it to?
-    num="$(sqlite3 "${gravityDBfile}" "SELECT COUNT(*) FROM ${list} WHERE ${sqlitekey} = \"${domain}\";")"
+    num="$(sqlite3 "${gravityDBfile}" "SELECT COUNT(*) FROM ${list} WHERE domain = \"${domain}\";")"
```

Comment:
Double quotes in SQL are usually used to wrap column or table names, whereas single quotes are used for strings (according to the SQL standards). SQLite appears to support using double quotes for strings, but we should try to stick to the standard.